loops-and-arrays.js

```diff
@@ -37,6 +49,12 @@ const numbers = [2, 4, 5, 29, 38];
 
 const squares = [30, 2, 8, 24, 11];
 
+for (let i = 0; i < squares.length; i++) {
+
+    squares[i] = Math.pow(squares[i], 3);
```

Comment:
Wow! je hebt iets dieper gegraven dan nodig was. Uitkomst is goed, maar voor nu was deze simpele aanpak voldoende :
`squares[i] = squares[i] * squares[i] * squares[i];`